connect/snapshot/Dockerfile

```diff
@@ -11,17 +11,32 @@ ENV DEBEZIUM_VERSION=$DEBEZIUM_VERSION \
 # Download the snapshot version of the connectors and then install them into the `$KAFKA_CONNECT_PLUGINS_DIR/debezium` directory...
 #
 RUN for CONNECTOR in {mysql,mongodb,postgres,sqlserver,oracle,db2,vitess}; do \
-    SNAPSHOT_VERSION=$(curl -fSL $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-connector-$CONNECTOR/$DEBEZIUM_VERSION/maven-metadata.xml | awk -F'<[^>]+>' '/<extension>tar.gz<\/extension>/ {getline; print $2}'); \
-    curl -fSL -o /tmp/plugin.tar.gz \
-                 $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-connector-$CONNECTOR/$DEBEZIUM_VERSION/debezium-connector-$CONNECTOR-$SNAPSHOT_VERSION-plugin.tar.gz &&\
+    SNAPSHOT_VERSION=$(curl --silent -fSL $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-connector-$CONNECTOR/$DEBEZIUM_VERSION/maven-metadata.xml | awk -F'<[^>]+>' '/<extension>tar.gz<\/extension>/ {getline; print $2; exit}'); \
+    echo "Downloading and installing debezium-connector-$CONNECTOR-$SNAPSHOT_VERSION-plugin.tar.gz ..." ; \
+    curl --silent -fSL -o /tmp/plugin.tar.gz \
+                 $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-connector-$CONNECTOR/$DEBEZIUM_VERSION/debezium-connector-$CONNECTOR-$SNAPSHOT_VERSION-plugin.tar.gz && \
+    echo "Extracting debezium-connector-$CONNECTOR-$SNAPSHOT_VERSION-plugin.tar.gz ..." && \
     tar -xzf /tmp/plugin.tar.gz -C $KAFKA_CONNECT_PLUGINS_DIR && \
+    echo "Successfully installed debezium-connector-$CONNECTOR-$SNAPSHOT_VERSION!" && \
     rm -f /tmp/plugin.tar.gz; \
-    done
+done
 
-RUN for PACKAGE in {scripting}; do \
-    SNAPSHOT_VERSION=$(curl -fSL $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-$PACKAGE/$DEBEZIUM_VERSION/maven-metadata.xml | awk -F'<[^>]+>' '/<extension>tar.gz<\/extension>/ {getline; print $2}'); \
-    curl -fSL -o /tmp/package.tar.gz \
+RUN for PACKAGE in {scripting,}; do \
+    SNAPSHOT_VERSION=$(curl --silent -fSL $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-$PACKAGE/$DEBEZIUM_VERSION/maven-metadata.xml | awk -F'<[^>]+>' '/<extension>tar.gz<\/extension>/ {getline; print $2; exit}'); \
+    echo "Downloading and installing debezium-$PACKAGE-$SNAPSHOT_VERSION.tar.gz ..." ; \
+    curl --silent -fSL -o /tmp/package.tar.gz \
                  $MAVEN_OSS_SNAPSHOT/io/debezium/debezium-$PACKAGE/$DEBEZIUM_VERSION/debezium-$PACKAGE-$SNAPSHOT_VERSION.tar.gz &&\
+    echo "Extracting debezium-$PACKAGE-$SNAPSHOT_VERSION.tar.gz ..." && \
     tar -xzf /tmp/package.tar.gz -C $EXTERNAL_LIBS_DIR && \
+    echo "Successfully installed debezium-$PACKAGE-$SNAPSHOT_VERSION!" && \
     rm -f /tmp/package.tar.gz; \
-    done
+done
+
+RUN for PACKAGE in {connect-rest-extension,}; do \
```

Comment:
Would it be possible to merge the two `extension` loops? IIUC they only differ in the filename suffix.

Reply:
No, they also differ in what they evaluate from the `maven-metadata.xml`.